projects/plugins/social/src/class-note.php

```diff
@@ -104,4 +105,26 @@ public static function may_be_flush_rewrite_rules( $force = false ) {
 	public static function delete_rewrite_rules_option() {
 		delete_option( self::JETPACK_SOCIAL_REWRITE_RULES_LAST_FLUSHED_AT );
 	}
+
+	/**
+	 * Use the_title hook so we show the social note's exceprt in the post list view.
+	 *
+	 * @param array $title The title of the post, which we have set to be an empty string for Social Notes.
+	 * @param array $post_id The Post ID.
+	 */
+	public static function override_empty_title( $title, $post_id ) {
+		if ( is_admin() && 'edit.php' === $GLOBALS['pagenow'] ) {
+			// Get the post type
+			$post_type = get_post_type( $post_id );
+
+			// Check if the post type doesn't have a title
+			if ( $post_type === self::JETPACK_SOCIAL_NOTE_CPT ) {
+				// Return permalink instead of title
+				return wp_strip_all_tags( wp_trim_excerpt( get_post_field( 'post_content', $post_id ) ) );
```

Comment:
By default wp_trim_excerpt returns 55 words https://developer.wordpress.org/reference/functions/wp_trim_excerpt/ I think we should probably reduce that for this title. Maybe 10?